tutorials/pretrain.md

```diff
@@ -36,8 +36,65 @@ litgpt pretrain \
 ```
 
 
+&nbsp;
+## Pretrain on custom data
+
+The simplest way to get started with pretraining on custom data is by using the `TextFiles` data module, which lets you pretrain a dataset from a folder containing plain text files. 
+
+For instance, assume you stored a number of text files in a `custom_pretraining_dataset` folder (we recommend avoiding small files and concatenating them to files of at least 50 Mb for efficiency):
```

Comment:
Yes it may be the simplest way but for actual pretraining from scratch on a large dataset that won't be practical. Since text data is highly compressible, it is often stored in compressed format, and often in file formats where documents can be loaded row by row without having to load entire files at once. So I would only recommend this simplified format for continued pretraining where it is feasible to store the data in plain text files due to the limited size.

Reply:
You are totally right. I added a Warning! to both sections to make this clear.